doc/1.4/language.md

```diff
@@ -1644,6 +1638,58 @@ and inline methods remain mainly for compatibility reasons.
 In DML 1.4, methods can be `exported` using the
 [`export` declaration](#export-declarations).
 
+### Method Qualifiers
+There are a number of qualifiers that can be applied to method declarations.
+These are specified through a space delimited list immediately before the
+`method` keyword in a declaration, and any method declaration overriding another
+must share the same qualifiers as the overridden method declaration. Method
+qualifiers can be used for both regular methods as well as
+[`shared` methods](#shared-methods). They can't be used for
+[`inline` methods](#inline-methods).
+
+#### Independent Methods
+Methods that do not rely on the particular instance of the device model may
+be declared `independent`:
+```
+independent method m(...) -> (...) {...}
+```
+[Exported](#export-declarations) independent methods do not have the input
+parameter corresponding to the device instance, allowing them to be called
+in greater number of contexts. The body of independent methods may not contain
+statements or expressions that rely on the device instance in any way; for
+example, `session` or `saved` variables may not be referenced, `after` and `log`
+statements may not be used, and non-`independent` may not be called.
```

Comment:
`and non-`independent` may not be called.` should be `and non-`independent` methods may not be called.`